incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -2098,21 +2285,21 @@ private void sendWillSignal(
                             .value(WILL_SIGNAL_NAME.value(), 0, WILL_SIGNAL_NAME.length())))
                 .build();
 
-            final MqttWillSignalFW willSignal =
-                mqttWillSignalRW.wrap(willSignalBuffer, 0, willSignalBuffer.capacity())
-                    .clientId(delegate.clientId)
-                    .delay(delegate.delay)
-                    .deliverAt(supplyTime.getAsLong() + delegate.delay)
-                    .lifetimeId(delegate.lifetimeId)
-                    .willId(delegate.willId)
-                    .instanceId(instanceId.instanceId())
+            final MqttSessionSignalFW willSignal =
+                mqttSessionSignalRW.wrap(sessionSignalBuffer, 0, sessionSignalBuffer.capacity())
+                    .will(w -> w.clientId(delegate.clientId)
+                        .delay(delegate.delay)
+                        .deliverAt(supplyTime.getAsLong() + delegate.delay)
+                        .lifetimeId(delegate.lifetimeId)
+                        .willId(delegate.willId)
+                        .instanceId(instanceId.instanceId()))
```

Comment:
`.build()` should be aligend with `.instanceId()` above it.
Also suggest, moving `.clientId(...)` to next line for readability.